src/frontend/src/flows/manage/authenticatorsSection.ts

```diff
@@ -109,6 +109,32 @@ export const authenticatorsSection = ({
     </aside>`;
 };
 
+export const tempKeysSection = ({
+  authenticators: authenticators_,
+}: {
+  authenticators: Authenticator[];
+}): TemplateResult => {
+  const authenticators = dedupLabels(authenticators_);
+
+  return html` <aside class="l-stack c-card c-card--narrow">
+    <div class="t-title t-title--complications">
+      <h2 class="t-title">Temporary Keys</h2>
+    </div>
+
+    <p style="max-width: 30rem;" class="t-paragraph t-lead">
+      These keys are stored in browser cache, clearing your browser storage will
```

Comment:
i18n (generally speaking for any text provideed or modified in this PR)?

or this will be extracted later on?

Reply:
I don't want to change more than I need to, so I kept to the way that the `authenticatorsSection` works (which also does not have copy extracted).

This should definitely be extracted later.